README.rst

```diff
@@ -290,6 +292,12 @@ but you can add more too by using the ``fasttest`` special variable.
 
 See the Testing_ section for more details.
 
+The ``example`` target will compile every ``.d`` file found under the
```

Comment:
I think wxamples should get their own section in the documentation. I would only mention what is the target for in here and refer to the section for details, where the exclusions and a more general reasoning why this exists it is given.